build.gradle

```diff
@@ -78,3 +78,35 @@ test {
     }
 }
 
+private def setBrowserStackSystemProperties(requiredOsKey, requiredDeviceName) {
+    loadProperties()
+    println "Looking for BrowserStack config: [$requiredOsKey - $requiredDeviceName]"
+    config.eachWithIndex { def osTests, int i ->
```

Comment:
I don't think you need to use `eachWithIndex` here, as you're not using the index (that I can see). You can just use `each`: http://grails.asia/groovy-each-examples

Reply:
```
config.each { osTests ->
```

Reply:
Good point, have changed.